app/EvidencioAPI.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App;
+
+class EvidencioAPI
+{
+  /*
+   * Helper method to perform an API call
+   * @param $path URL to the call without leading "https://www.evidencio.com/api/"
+   * @param $params list of parameters as a key-value array
+   * @return API response converted into a PHP variable
+   * @throws JsonDecodeException if JSON response could not be decoded
+   * Other possible exceptions are listed in Guzzle's documentation
+   */
+  private static function fetch($path, $params = [])
```

Comment:
It is considered good practise to add argument type declarations and return type declarations for all functions.